README.md

```diff
@@ -3,7 +3,7 @@
 [![Anaconda-Server Badge](https://anaconda.org/scipp/essimaging/badges/version.svg)](https://anaconda.org/scipp/essimaging)
 [![License: BSD 3-Clause](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](LICENSE)
 
-# essimaging
+# Essimaging
```

Comment:
```suggestion
# ESSimaging
```

Reply:
This should have been fixed by changing the copier answer to `prettyname`.